.eslintrc

```diff
@@ -1,592 +1,7 @@
-			"error"
-		],
-		"no-shadow": [
-			"off"
-		],
-		"no-shadow-restricted-names": [
-			"error"
-		],
-		"no-spaced-func": [
-			"error"
-		],
-		"no-sync": [
-			"off"
-		],
-		"no-ternary": [
-			"off"
-		],
-		"no-throw-literal": [
-			"error"
-		],
-		"no-trailing-spaces": [
-			"error"
-		],
-		"no-undef-init": [
-			"error"
-		],
-		"no-undefined": [
-			"off"
-		],
-		"no-underscore-dangle": [
-			"off"
-		],
-		"no-unmodified-loop-condition": [
-			"error"
-		],
-		"no-unneeded-ternary": [
-			"error"
-		],
-		"no-unused-expressions": [
-			"warn",
-			{
-				"allowShortCircuit": true,
-				"allowTernary": true
-			}
-		],
-		"no-use-before-define": [
-			"off"
-		],
-		"no-useless-call": [
-			"error"
-		],
-		"no-useless-concat": [
-			"error"
-		],
-		"no-useless-constructor": [
-			"error"
-		],
-		"no-useless-escape": [
-			"error"
-		],
-		"no-void": [
-			"error"
-		],
-		"no-warning-comments": [
-			"off"
-		],
-		"no-whitespace-before-property": [
-			"error"
-		],
-		"no-with": [
-			"error"
-		],
-		"object-curly-spacing": [
-			"error",
-			"always"
-		],
-		"object-shorthand": [
-			"off"
-		],
-		"one-var": [
-			"off"
-		],
-		"one-var-declaration-per-line": [
-			"error"
-		],
-		"operator-assignment": [
-			"error",
-			"always"
-		],
-		"operator-linebreak": [
-			"warn",
-			"before"
-		],
-		"padded-blocks": [
-			"off"
-		],
-		"prefer-arrow-callback": [
-			"off"
-		],
-		"prefer-const": [
-			"off"
-		],
-		"prefer-reflect": [
-			"off",
-			{
-				"exceptions": [
-					"call"
-				]
-			}
-		],
-		"prefer-rest-params": [
-			"off"
-		],
-		"prefer-spread": [
-			"off"
-		],
-		"prefer-template": [
-			"off"
-		],
-		"quote-props": [
-			"off"
-		],
-		"quotes": [
-			"error",
-			"single"
-		],
-		"radix": [
-			"off"
-		],
-		"require-jsdoc": [
-			"off"
-		],
-		"require-yield": [
-			"error"
-		],
-		"semi": [
-			"error"
-		],
-		"semi-spacing": [
-			"error"
-		],
-		"sort-imports": [
-			"off"
-		],
-		"sort-vars": [
-			"off"
-		],
-		"space-before-blocks": [
-			"error",
-			"always"
-		],
-		"space-before-function-paren": [
-			"error",
-			{
-				"anonymous": "always",
-				"named": "never"
-			}
-		],
-		"space-in-parens": [
-			"error",
-			"never"
-		],
-		"space-infix-ops": [
-			"error"
-		],
-		"space-unary-ops": [
-			"error",
-			{
-				"nonwords": false,
-				"overrides": {}
-			}
-		],
-		"spaced-comment": [
-			"error"
-		],
-		"strict": [
-			"off"
-		],
-		"template-curly-spacing": [
-			"error"
-		],
-		"valid-jsdoc": [
-			"error",
-			{
-				"requireParamDescription": false,
-				"requireReturn": false,
-				"requireReturnDescription": false,
-				"requireReturnType": false
-			}
-		],
-		"vars-on-top": [
-			"warn"
-		],
-		"wrap-iife": [
-			"error"
-		],
-		"wrap-regex": [
-			"error"
-		],
-		"yield-star-spacing": [
-			"error"
-		],
-		"yoda": [
-			"error"
-		]
-	}
+  "extends": [ "axway/env-node", "axway/+mocha" ],
+  rules: {
+    'security/detect-child-process': 0,
```

Comment:
@jareddlc added overrides for the recommended child_process and regex rules.  MUCH prefer this being handled in the .eslintrc than inline.